gensrc/thrift/AgentService.thrift

```diff
@@ -150,6 +150,7 @@ struct TCreateTabletReq {
     23: optional i64 time_series_compaction_goal_size_mbytes = 1024
     24: optional i64 time_series_compaction_file_count_threshold = 2000
     25: optional i64 time_series_compaction_time_threshold_seconds = 3600
+    26: optional i64 time_series_compaction_empty_rowsets_threshold = 5
```

Comment:
Is it necessary to config in table level?

Reply:
The empty rowsets for different tables may vary，due to various configurations of the tables